thirdeye-spi/src/main/java/ai/startree/thirdeye/spi/api/RcaInvestigationApi.java

```diff
@@ -115,13 +114,4 @@ public RcaInvestigationApi setUpdatedBy(final UserApi updatedBy) {
     this.updatedBy = updatedBy;
     return this;
   }
-
-  public AuthorizationConfigurationApi getAuth() {
```

Comment:
Same comment as above. Even if backend doesn't rely on rca entity for auth, we don't need to push the onus on the UI to figure that out.